content/en/docs/distributions/gke/customizing-gke.md

```diff
@@ -156,78 +157,131 @@ options:
 To see which accelerators are available in each zone, run the following
 command:
 
-```
+```bash
 gcloud compute accelerator-types list
 ```
  
-To disable node-autoprovisioning, edit content for `${KF_DIR}/common/cluster/upstream/cluster.yaml`, set 
-[`enabled`](https://github.com/kubeflow/manifests/blob/4d2939d6c1a5fd862610382fde130cad33bfef75/gcp/deployment_manager_configs/cluster-kubeflow.yaml#L73) 
-to `false`:
+Create the [ContainerNodePool](https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containernodepool) resource adopting GPU, for exmaple, create a new file `containernodepool-gpu.yaml` file and fulfill the value below:
 
 ```
-    ...
-    clusterAutoscaling:
-      enabled: false
-      autoProvisioningDefaults:
-    ...
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerNodePool
+metadata:
+  labels:
+    kf-name: KUBEFLOW-NAME # {"$kpt-set":"name"}
+  name: containernodepool-gpu
+  namespace: KF-PROJECT # {"$kpt-set":"project"}
+spec:
+  location: LOCATION # {"$kpt-set":"location"}
+  initialNodeCount: 1
+  autoscaling:
+    minNodeCount: 0
+    maxNodeCount: 5
+  nodeConfig:
+    machineType: n1-standard-4
+    diskSizeGb: 100
+    diskType: pd-standard
+    preemptible: true
+    oauthScopes:
+    - "https://www.googleapis.com/auth/logging.write"
+    - "https://www.googleapis.com/auth/monitoring"
+    - "https://www.googleapis.com/auth/devstorage.read_only"
+    guestAccelerator:
+    - type: "nvidia-tesla-k80"
+      count: 1
+    metadata:
+      disable-legacy-endpoints: "true"
+  management:
+    autoRepair: true
+    autoUpgrade: true
+  clusterRef:
+    name: KUBEFLOW-NAME # {"$kpt-set":"name"}
+    namespace: KF-PROJECT # {"$kpt-set":"project"}
 ```
 
-Then create the [ContainerNodePool](https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containernodepool) resource adopting GPU, for exmaple:
+Note that the `metadata:name` should be unique in your Kubeflow project. Because management cluster uses this as ID and your Google Cloud project as namespace to identify a nodepool.
```

Comment:
the `metadata:name` should be -> the `metadata:name` must be
Because management -> Because the management
as namespace -> as a namespace
nodepool -> node pool